packages/react-composites/src/composites/ChatComposite/file-sharing/FileUpload.ts

```diff
@@ -174,34 +167,34 @@ export class FileUpload implements FileUploadManager, ObservableFileUpload {
  * Events emitted by the FileUpload class.
  * @beta
  */
-export type FileUploadEvents = 'uploadProgressChange' | 'uploadComplete' | 'uploadFail';
+type FileUploadEvents = 'uploadProgressChange' | 'uploadComplete' | 'uploadFail';
```

Comment:
Don't we need to export these event types and listeners anymore?

Reply:
Not anymore, no exported interface is using these types and hence we can keep them internal